src/pages/contact.js

```diff
@@ -2,10 +2,11 @@ import React from "react"
 
 const Contact = () => {
     return (
-        <div>
+        <div className="General">
```

Comment:
CSS classes are per convention kebab-case. This means they should be lower case, and with a dash (-) in between each word.
There are some people who name CSS classes **for components** with upper case first letter (like .App, or in this case, .Contact). But this is not universal.